com.unity.render-pipelines.universal/Editor/RendererFeatures/NewRendererFeature.cs.txt

```diff
@@ -23,8 +23,8 @@ public class #SCRIPTNAME# : ScriptableRendererFeature
         {
         }
 
-        /// Cleanup any allocated resources that were created during the execution of this render pass.
-        public override void FrameCleanup(CommandBuffer cmd)
+        // Cleanup any allocated resources that were created during the execution of this render pass.
```

Comment:
add docs as according to xml docs: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/